core/app.js

```diff
@@ -296,12 +296,18 @@ app.use(_.get("/employees",function *(){
   var isAdmin = yield auth.isAdmin(this.session.user);
   var client = db();
   var allAdmins = yield client.query("SELECT netid, email FROM users WHERE admin = TRUE;");
+  var names = []
+  for(var x = 0; x < allAdmins.rows.length; x++){
+    var db_id = (allAdmins.rows[x]).netid;
+    var data = yield user.ldapInfo(db_id);
+    names.push(data[db_id].cn);
+  }
```

Comment:
nope! looks good to me. sorry for the bad information
On Apr 6, 2016 5:30 PM, "Stephen Thomas" <notifications@github.com> wrote:

> In core/app.js
> <https://github.com/BYU-ODH/hlrdesk/pull/355#discussion_r58785431>:
>
> > @@ -296,12 +296,18 @@ app.use(_.get("/employees",function *(){
> >    var isAdmin = yield auth.isAdmin(this.session.user);
> >    var client = db();
> >    var allAdmins = yield client.query("SELECT netid, email FROM users WHERE admin = TRUE;");
> > +  var names = []
> > +  for(var x = 0; x < allAdmins.rows.length; x++){
> > +    var db_id = (allAdmins.rows[x]).netid;
> > +    var data = yield user.ldapInfo(db_id);
> > +    names.push(data[db_id].cn);
> > +  }
>
> var netids = allAdmins.rows.map(function (a){return a.netid});
> var ldapDetails = yield user.ldapInfo.apply(null, netids);
> var names = []
> for (var key in ldapDetails) {
> names.push(ldapDetails[key].cn);
> }
>
> This code ^^^ is working. The variable ldapDetails is a hash element and
> wasn't accepting .map. Do you have any suggestions beyond what I've done?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/BYU-ODH/hlrdesk/pull/355/files/a65a1727b9d3d0ba144cd6d339ad5255a3a5a072#r58785431>
>